sdk/search/Azure.Search.Documents/README.md

```diff
@@ -317,7 +319,7 @@ SearchOptions options = new SearchOptions
     // Filter to only Rating greater than or equal our preference
     Filter = SearchFilter.Create($"Rating ge {stars}"),
     Size = 5, // Take only 5 results
-    OrderBy = { "Rating desc" } // Sort by Rating from high to low
+   // OrderBy = { "Rating desc" } // Sort by Rating from high to low
```

Comment:
Given this shows up in a sample, I'd remove this line entirely. In a sample, it seems confusing.

Reply:
[This](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2101992&view=logs&j=283d73df-0aa6-5cae-0d63-20b601dc19a3&t=40ae92eb-816f-5cde-3fce-f7b41ef072b5&l=562) test case is passing locally(in live, record and playback all three modes) but failing in nightly pipeline. So I was just trying to debug and commented out this line for testing purposes. 

This line is now uncommented.